File: CF/1300-1399/1372B.cpp
// submission link: https://codeforces.com/contest/1372/submission/271861511

#include<cmath>
#include<iostream>

void solve(){
    int n;
    std::cin >> n;
    for (int i = 2; i < sqrt(n) + 1; i ++){
        if (n % i == 0) {
            int k = n / i;
            std::cout << k << ' ' << n - k << std::endl;
            return;
        }
    }
    std::cout << 1 << " " << n - 1 << std::endl;
    return;
}

int main(){
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(0);
    std::cout.tie(0);
    int check;
    std::cin >> check;
    while (check --) {
        solve();
    }
    return 0;
}

File: CF/1300-1399/1372B.py
# submission link: https://codeforces.com/contest/1372/submission/271860753
from math import sqrt
import sys
input = lambda: sys.stdin.readline().strip()
check = int(input())
'''
def LCD(a: int, b: int) -> int:
    c = b % a
    if c == 0:
        return a
    if c > a:
        a, c = c, a
    return LCD(c, a)

def LCM(a: int, b: int) -> int:
    return a * b // LCD(a, b)
'''

for _ in range(check):
    n = int(input())
    a = 1; b = n - 1
    k = 0
    for i in range(2, int(sqrt(n)) + 1):
        if n % i == 0:
            k = n // i 
            break
    if k:
        a = k
        b = n - k
            
    print(a, b)
